internal/service/ec2/vpc_ipam_preview_next_cidr_test.go

```diff
@@ -0,0 +1,119 @@
+package ec2_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-aws/internal/acctest"
+)
+
+func TestAccVPCIpamPreviewNextCidr_ipv4Basic(t *testing.T) {
+	resourceName := "aws_vpc_ipam_preview_next_cidr.test"
+	netmaskLength := "28"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acctest.PreCheck(t) },
+		ErrorCheck:   acctest.ErrorCheck(t, ec2.EndpointsID),
+		Providers:    acctest.Providers,
+		CheckDestroy: nil,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccVPCIpamPreviewNextCidrIpv4Basic(netmaskLength),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet(resourceName, "cidr"),
+					resource.TestCheckResourceAttrSet(resourceName, "id"),
+					resource.TestCheckResourceAttrPair(resourceName, "ipam_pool_id", "aws_vpc_ipam_pool.test", "id"),
+					resource.TestCheckResourceAttr(resourceName, "netmask_length", netmaskLength),
+				),
+			},
+			// Errors with the import
```

Comment:
these can be removed per comment in resource schema

Reply:
Resolved in 9919d9371f86657f6cfa4e8527cbeb8830098c44